lib/tasks/import.rake

```diff
@@ -3,4 +3,40 @@ namespace :import do
   task countries_geometries: :environment do
     CountriesGeometryImporter.import
   end
+
+  desc "Import statistics files for new release"
+  task new_release_statistics: :environment do
+
+    # To be run AFTER deploying a new release 
+    # Function:
+      # 1. Checks an S3 bucket exists for the date constants provided in config/initializers/constants.rb
```

Comment:
This task will be useful to avoid running every single command to import those stats. But I don't fully follow the check on the bucket here, as these files are csv provided separately from the gdb

Reply:
you're right, it doesn't belong here